pkg/isoeditor/rhcos.go

```diff
@@ -87,8 +93,8 @@ func embedInitrdPlaceholders(extractDir string) error {
 	return nil
 }
 
-func fixTemplateConfigs(rootFSURL, extractDir string) error {
-	availableGrubPaths := []string{"EFI/redhat/grub.cfg", "EFI/fedora/grub.cfg"}
+func fixGrubConfig(rootFSURL, extractDir string) error {
+	availableGrubPaths := []string{"EFI/redhat/grub.cfg", "EFI/fedora/grub.cfg", "boot/grub/grub.cfg"}
```

Comment:
So iiuc, no EFI folder in ppc/s390x images?

Reply:
There's no `grub.cfg` in x390x at all, but it's just in a different place for ppc64le

Reply:
Yes, ppc64le and s390x don't use EFI